src/script/lua_api/l_object.cpp

```diff
@@ -1248,6 +1248,34 @@ int ObjectRef::l_get_inventory_formspec(lua_State *L)
 	return 1;
 }
 
+// set_inventory_formspec(self, formspec)
```

Comment:
Helpful comment. Needs removal or adjusting.

Reply:
all functions are documented in this manner, leave it like this